scripts/helmAcs.sh

```diff
@@ -237,13 +236,15 @@ share:
     initialDelaySeconds: 420
 registryPullSecrets: quay-registry-secret" > acs_install_values.yaml
 
-    helm install alfresco-incubator/alfresco-content-services --version 1.1.6 -f acs_install_values.yaml --name $ACS_RELEASE --namespace=$DESIREDNAMESPACE
+    CHART_VERSION=1.1.6
```

Comment:
Great work @mmuller88! But may be it will be more handy if this is set as a CFN template parameter so in real-time helm upgrades can be performed via cloudformation UPDATE_STACK.